libdnf/dnf-sack-private.hpp

```diff
@@ -63,4 +64,8 @@ Queue       *dnf_sack_get_installonly       (DnfSack    *sack);
 void         dnf_sack_set_running_kernel_fn (DnfSack    *sack,
                                              dnf_sack_running_kernel_fn_t fn);
 
+DnfPackage  *dnf_sack_add_cmdline_package_flags   (DnfSack *sack,
+                            const char *fn,
+                            const int flags=REPO_REUSE_REPODATA|REPO_NO_INTERNALIZE);
```

Comment:
Hmm, see #395 for previous discussions about trying to keep the API pure C. This *is* a private header, though making use of it in the Python bindings makes it that much harder to stay compatible, right?

If we go with the "rpm-ostree should explicitly specify those flags" way, maybe we can just revert `dnf_sack_add_cmdline_package` to just use `REPO_REUSE_REPODATA|REPO_NO_INTERNALIZE` and add a new public `dnf_sack_add_cmdline_package_flags` that takes flags parameters?

Reply:
dnf_sack_add_cmdline_package_flags moved from private header into public and rewritten to pure C, not using default parameter anymore.